cmd/goreleaser/internal/configure.go

```diff
@@ -392,18 +409,22 @@ func (d *distribution) BuildProject() config.Project {
 		builds = append(builds, buildConfig.Build(d.name))
 	}
 
+	env := []string{
+		"COSIGN_YES=true",
+		"LD_FLAGS=-s -w",
+		"BUILD_FLAGS=-trimpath",
+	}
+	if !d.enableCgo {
```

Comment:
Should this print out a warning or return an error, if the project is `ebpfProfilerDist`?

Reply:
Hmm I'm not sure I see the point. Checking for project name here kind of defeats the purpose of having an option.
In that case, let's not have this config, and use a switch statement with project names (I don't think that'd be a good idea).

Maybe I don't understand what you want though.